www/front_src/public/index.html

```diff
@@ -33,4 +33,22 @@
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
   </body>
+  <script type="text/javascript">
+    let platformData = localStorage.getItem('centreonPlatformData');
+    if (platformData !== null) {
+      platformData = JSON.parse(platformData);
+      // Pendo.io
+      (function(apiKey, platformData){
+      (function(p,e,n,d,o){var v,w,x,y,z;o=p[d]=p[d]||{};o._q=o._q||[];
+      v=['initialize','identify','updateOptions','pageLoad','track'];for(w=0,x=v.length;w<x;++w)(function(m){
+          o[m]=o[m]||function(){o._q[m===v[0]?'unshift':'push']([m].concat([].slice.call(arguments,0)));};})(v[w]);
+          y=e.createElement(n);y.async=!0;y.src='https://cdn.eu.pendo.io/agent/static/'+apiKey+'/pendo.js';
+          z=e.getElementsByTagName(n)[0];z.parentNode.insertBefore(y,z);})(window,document,'script','pendo');
+
+          // Call this whenever information about your visitors becomes available
+          // Please use Strings, Numbers, or Bools for value types.
+          pendo.initialize(platformData);
+      })('b06b875d-4a10-4365-7edf-8efeaf53dfdd', platformData);
```

Comment:
How does the apiKey works in Pendo ?

Reply:
It is used to send to data to centreon account in the previous call. 
`y.async=!0;y.src='https://cdn.eu.pendo.io/agent/static/'+apiKey+'/pendo.js';`
It's like an unique id, but not an authentication token or something.

Initially i was thinking of not put the key in raw and discuss with @lpinsivy to put it in a macro in centreon-build to avoid key in clear. But discussing it with @kduret he tolds me the fact that a macro will not work for sources installations, and the API key is still sent into an asynchronous request and is reachable from your browser.